src/plugins/layout.ts

```diff
@@ -96,40 +96,42 @@ export default class Layout implements IPlugin {
     while (this.domDiscoverQueue.length > 0 && getTimestamp(true) < yieldTime) {
       let index = this.domDiscoverQueue.shift();
       let shadowNode = this.shadowDom.getShadowNode(index);
-      let oldLayoutState = shadowNode.layout;
-      let layoutState = createLayoutState(shadowNode.node, this.shadowDom);
-      let originalProperties = this.originalProperties[index];
-
-      if (layoutState.tag !== IgnoreTag) {
-
-        // If we have any original properties recorded for this node, adjust layoutState with those values
-        if (originalProperties) {
-          switch (shadowNode.node.nodeType) {
-            case Node.TEXT_NODE:
-              (layoutState as ITextLayoutState).content = originalProperties.characterData;
-              break;
-            case Node.ELEMENT_NODE:
-              let originalAttributes = (layoutState as IElementLayoutState).attributes;
-              for (let attr in originalAttributes) {
-                if (originalAttributes.hasOwnProperty(attr)) {
-                  (layoutState as IElementLayoutState).attributes[attr] = originalAttributes[attr];
+
+      if (shadowNode) {
```

Comment:
We should explicitly log something in an else clause then so we have a way to investigate further

---
In reply to: [124159155](https://github.com/Microsoft/clarity-js/pull/20#discussion_r124159155) [](ancestors = 124159155)